### ISSUE TITLE
Relation Features Not Working In App

### ISSUE THREAD

**Issue by ekingapex** [MerginMaps/input#2551]
Our team is running into a critical issue with relation items in the app as well. Whenever creating child items, it is duplicating every child item created in the app on every parent feature. IE, creating Parent1 and adding Child1 and Child2 on Parent1, and then creating Parent2, Child1 and Child2 will show up linked to Parent 2, and adding further children will continue to add them to all parent features. The features were working correctly before the update and are working correctly in QGIS, so this issue is specific to the app. This issue is preventing features using 1:N relations from being viable through the app, as there is no way to discern which child features are linked to each parent feature.

**Comment by alexbruy** [comment]
Can you share your project with the `support` account and tell its name?

**Comment by ekingapex** [comment]
With the change to workspaces, I am no longer able to search for users other than by email address to add them to a project; is this a known issue?

I have instead made an example project and made it public, which has a parent feature layer and two example feature layers, one for data input and one for photographs. I’ve included two features already created within the app to demonstrate what is happening; child1 and child2 were created under Parent1, child3 under Parent2; creating further parent and child features/photographs will continue to demonstrate how it is duplicating links on the child features and tying them all back to Parent1. This should hopefully make it easier to narrow down what the source of the problem may be. The project is under my workspace EKING, named RelationExample.

https://app.merginmaps.com/projects/eking/RelationExample/tree


Thank you for your time,
Evan King
Royal Treasury Enterprises, LLC on behalf of Apex Advanced Technology
770-601-7665

________________________________

**Comment by ekingapex** [comment]
Thank you for the work around, we will implement this while the bug fix is being worked on. I did notice a peculiar side effect from the work around; it seems to add an empty linked child feature that make a “+1 More” button show up in the parent form where the user would add child items. It doesn’t seem to affect anything and tapping on it just brings up a blank form, but I figured I would mention it in case it is tied to anything that might affect the bug fix.

Thank you for your time,
Evan King
Royal Treasury Enterprises, LLC on behalf of Apex Advanced Technology
770-601-7665
